invenio.cfg

```
@@ -333,3 +334,56 @@ CDS_EOS_OFFLOAD_X509_KEY_PATH = ""
 CDS_EOS_OFFLOAD_REDIRECT_BASE_PATH = ""
 
 RDM_PERMISSION_POLICY = CDSRDMRecordPermissionPolicy
+
+# Custom fields
+RDM_NAMESPACES = {
+    # CERN
+    "cern": "https://greybook.cern.ch/",
+}
+
+RDM_CUSTOM_FIELDS = [
+    VocabularyCF(
+        name="cern:experiment",
+        vocabulary_id="experiments",
+        dump_options=True, 
+        multiple=False,
+    ),
+    VocabularyCF(
+        name="cern:department",
+        vocabulary_id="departments",
+        dump_options=True,
+        multiple=False,
+    ),
+]
+
+RDM_CUSTOM_FIELDS_UI = [
+    {
+        "section": _("CERN Information"),
+        "fields": [
+            dict(
+                field="cern:experiment",
+                ui_widget="Dropdown",
+                props=dict(
+                    label="Experiments",
+                    icon="lab",
+                    description="You should fill this field with one of the experiments e.g IS, EM etc.",
```

```suggestion
                    description="You should fill this field with one of the experiments e.g ATLAS, CMS etc.",
```
maybe it is better to use some of the bigger experiments in the example?